torch/nn/parallel/distributed.py

```diff
@@ -1111,29 +1111,29 @@ def join(
 
         Example::
 
-          >>>  import torch
-          >>>  import torch.distributed as dist
-          >>>  import os
-          >>>  import torch.multiprocessing as mp
-          >>>  import torch.nn as nn
-          >>>  # On each spawned worker
-          >>>  def worker(rank):
-          >>>      dist.init_process_group("nccl", rank=rank, world_size=2)
-          >>>      torch.cuda.set_device(rank)
-          >>>      model = nn.Linear(1, 1, bias=False).to(rank)
-          >>>      model = torch.nn.parallel.DistributedDataParallel(
-          >>>          model, device_ids=[rank], output_device=rank
-          >>>      )
-          >>>      # Rank 1 gets one more input than rank 0.
-          >>>      inputs = [torch.tensor([1]).float() for _ in range(10 + rank)]
-          >>>      with model.join():
-          >>>          for _ in range(5):
-          >>>              for inp in inputs:
-          >>>                  loss = model(inp).sum()
-          >>>                  loss.backward()
-          >>>  # Without the join() API, the below synchronization will hang
-          >>>  # blocking for rank 1's allreduce to complete.
-          >>>  torch.cuda.synchronize(device=rank)
+            >>> import torch
```

Comment:
The code examples in the other docstrings in the file used a 4-space indent with respect to `Example::` and had only one space after the `>>>`, so I thought to make it uniform.